### DISCUSSION TITLE
Constant value not available with define function

### DISCUSSION THREAD

**Discussion by stevvvv** [bootscore/bootscore#780] [Q&A]
When I used the define function to set a constant in the wp-config.php or function.php file they are not available within the template. When I use a a different template the constant value is available.

Error message
file_get_contents(#####): failed to open stream: No such file or directory in /data/sites/web/###wp-content/themes/bootscore-child/page-templates/###.php on line 50

I already disabled all my plugins, but no result. Any suggestions?

**Comment by crftwrk** [comment]
@justinkruit ?

**Comment by justinkruit** [comment]
I'm sorry for the late response @stevvvv. Could you fill this form and give us the ID? Then we can debug this on your environment. 
https://bootscore.me/private-info-form/
